dspace-api/src/main/java/org/dspace/submit/step/UploadStep.java

```diff
@@ -614,7 +614,7 @@ public int processUploadFile(Context context, HttpServletRequest request,
                 if (configurationService.getBooleanProperty("submission-curation.virus-scan"))
                 {
                     Curator curator = new Curator();
-                    curator.addTask("vscan").curate(item);
+                    curator.addTask("vscan").curate(context, item.getHandle());
```

Comment:
I could argue that a non-breaking change needed for a bug fix is part of the bug fix, and that we should ignore the fact that it also introduces a new API feature that nobody asked for.